src/extension/background-script/actions/webbtc/getAddress.ts

```diff
@@ -0,0 +1,60 @@
+import { getTaprootAddressFromPrivateKey } from "~/common/lib/btc";
+import { decryptData } from "~/common/lib/crypto";
+import {
+  BTC_TAPROOT_DERIVATION_PATH,
+  BTC_TAPROOT_DERIVATION_PATH_REGTEST,
+  derivePrivateKey,
+  derivePublicKey,
+} from "~/common/lib/mnemonic";
+import state from "~/extension/background-script/state";
+import { BitcoinAddress, MessageGetAddress } from "~/types";
+
+const getAddress = async (message: MessageGetAddress) => {
+  try {
+    // TODO: is this the correct way to decrypt the mnmenonic?
+    const password = await state.getState().password();
+    if (!password) {
+      throw new Error("No password set");
+    }
+    const account = await state.getState().getAccount();
+    if (!account) {
+      throw new Error("No account selected");
+    }
+    if (!account.mnemonic) {
+      throw new Error("No mnemonic set");
+    }
+    const mnemonic = decryptData(account.mnemonic, password);
+    const settings = state.getState().settings;
+
+    const derivationPath =
+      settings.bitcoinNetwork === "bitcoin"
```

Comment:
wondering if this should be an account setting and not a global setting?

Reply:
It makes sense to me as an account setting. @reneaaron do you have any thoughts here? I think we just did it this way to keep it as simple as possible while still being able to change it without having to build the extension locally (e.g. with an environment variable)